.importjs.json

```diff
@@ -2,6 +2,7 @@
   {
     "environments": ["node"],
     "ignorePackagePrefixes": ["lodash."],
+    "logLevel": "debug",
```

Comment:
I think this is the first config option that I've come across where it made sense.